Kickstarter-iOS/Features/PledgedProjectsOverview/PPOContainerViewController.swift

```diff
@@ -20,6 +23,26 @@ public class PPOContainerViewController: PagedContainerViewController<PPOContain
       (.projectAlerts(.count(5)), ppoViewController),
       (.activityFeed(.dot), activitiesViewController)
     ])
+
+    let tabBarController = self.tabBarController as? RootTabBarViewController
+
+    self.ppoViewModel.$results.sink { [weak self] results in
+      let badge: TabBarBadge = results.values.count > 0 ? .count(results.values.count) : .none
```

Comment:
If a user has enough results that this paginates, won't the badge count be off? I think we should either use a `totalCount` which I think graphql gives us or just do a dot here for now and change it to a count once the backend updates to give us that count (not scoped for v1).

Reply:
Added support for the totalCount in https://github.com/kickstarter/ios-oss/pull/2161/commits/504282fedb6642d2565693aa0a1299428ee980cb